Resources/Prototypes/Entities/Objects/Weapons/Melee/stunprod.yml

```diff
@@ -0,0 +1,48 @@
+- type: entity
+  name: stun prod
+  parent: BaseItem
+  id: Stunprod
+  description: A stun prod for illegal incapacitation.
+  components:
+  - type: Sprite
+    sprite: Objects/Weapons/Melee/stunprod.rsi
+    layers:
+    - state: stunprod_off
+      map: [ "enum.ToggleVisuals.Layer" ]
+  - type: Stunbaton
+    energyPerUse: 70
+  - type: MeleeWeapon
+    damage:
+      types:
+        Blunt: 9
+    bluntStaminaDamageFactor: 1.5
```

Comment:
Shouldn't have a custom blunt override.

Also isn't this a 3-hit stun? 9 x 1.5 + 20 = 33.5

Reply:
> Also isn't this a 3-hit stun? 9 x 1.5 + 20 = 33.5

no, stamina dmg != normal dmg